### ISSUE TITLE
OpenMP collapse support

### ISSUE THREAD

**Issue by Chronum94** [serge-sans-paille/pythran#1219]
```python
#pythran export trap3d(float32[:, :, :])
def trap3d(arr):
    x, y, z = arr.shape

    trapint = 0.0
    #omp parallel for reduction(+:trapint)
    for i in range(x):
        for j in range(y):
            for k in range(z):
                trapint += arr[i, j, k]
    return trapint
```

While running this I realized that pythran doesn't support the Openmp collapse clause. Would it be possible to implement collapse?

This is a fairly artificial test, but I'm just using it as a proof of concept.

**Comment by diorcety** [comment]
What is the issue exactly? It seems to work on my side

**Comment by Chronum94** [comment]
If you add the `collapse(2)` clause to the above loops, it should fail.

**Comment by serge-sans-paille** [comment]
#1250 should (finally!) fix this issue. We're getting closer to OpneMP3 support :-)

**Comment by Chronum94** [comment]
I tried testing the new feature, but I don't have the bleeding edge gast version (0.2.2, I think?) since only 0.2.0 is on pip. Will be happy to test it as soon as I can.

**Comment by serge-sans-paille** [comment]
I just merged in #1250, can you confirm it fixes your issue?

**Comment by Chronum94** [comment]
Fixes issue. Beautiful!
